keps/sig-storage/3333-reconcile-default-storage-class/kep.yaml

```diff
@@ -40,5 +43,5 @@ disable-supported: true
 
 # The following PRR answers are required at beta release
 metrics:
-#  TBD
-#  - my_feature_metric
+  - retroactive_storageclass_fail
+  - retroactive_storageclass_success
```

Comment:
These don't match the metrics described above.

Reply:
Thanks, fixed.